sst/test/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+set(SIMENG_SST_TEST_SOURCES 
+    main.cc
+    src/tg0_llvm_assemble.cc
+    src/tg1_load_store.cc
+    src/tg2_cache_access.cc
+    src/tg3_request_split.cc
+    src/tg4_request_misaligned.cc
+    src/tg5_benchmarks.cc
```

Comment:
Is it possible to do `src/*`? This would avoid having to add new tests if any are created.

Also, renaming `src` to `test_files` or `test_source` would be more descriptive

Reply:
Understood, thanks